be/src/runtime/load_channel.h

```diff
@@ -85,6 +85,8 @@ class LoadChannel : public RefCountedThreadSafe<LoadChannel> {
 
     MemTracker* mem_tracker() { return _mem_tracker.get(); }
 
+    std::string get_span() { return Tracer::to_trace_parent(_span->GetContext()); }
```

Comment:
why not just return _span, trace_parent is not need in rpc or http request.

Reply:
Thank you so much, that will be better.